src/components/pages/Evaluation/DefineScopePage.svelte

```diff
@@ -156,9 +159,44 @@
       return available.indexOf(assertion.test.num) == -1;
     });
 
-    assertionsToRemove.forEach((assertion) => {
-      assertions.remove(assertion);
-    });
+    if(assertionsToRemove.length > 0){
+      let answeredCount = 0;
+      assertionsToRemove.forEach((assertion) => {
+        if(assertion.result.outcome.id != "earl:untested"){
+          answeredCount++;
+        }
+      });
+
+      if(answeredCount > 0){
+        if (confirm(TRANSLATED.DATA_LOSS_WARNING)) {
+          assertionsToRemove.forEach((assertion) => {
+            assertions.remove(assertion);
+          });
+        } else {
+          $scopeStore['WCAG_VERSION'] = oldwcag;
+          $scopeStore['CONFORMANCE_TARGET'] = oldtarget;
+        }
+      }else{
+        assertionsToRemove.forEach((assertion) => {
+          assertions.remove(assertion);
+        });
+      }
+    }
+    
   }  
 
+  let oldwcag = "";
+  let oldtarget = "";
+  onMount(() => {
+    oldwcag = document.getElementById("wcag_version").value;
+    oldtarget = document.getElementById("conformance_target").value;
+
+    document.getElementById("wcag_version").onfocus=function() {
+       oldwcag = this.value;
```

Comment:
we should bind this listener in the template I think, this will be difficult to understand when we look at it again in a few months time